hypertrace-ingester/build.gradle.kts

```diff
@@ -67,13 +74,34 @@ fun createCopySpec(projectName: String, serviceName: String): CopySpec {
 
 tasks.register<Copy>("createCopySpecForSubJob") {
   with(
-      createCopySpecForSubJob("hypertrace-view-generator", "hypertrace-view-generator")
+      createCopySpecForSubJob("hypertrace-view-generator", "hypertrace-view-generator", "main")
   ).into("./build/resources/main/configs/")
 }
 
-fun createCopySpecForSubJob(projectName: String, serviceName: String): CopySpec {
+fun createCopySpecForSubJob(projectName: String, serviceName: String, srcFolder: String): CopySpec {
   return copySpec {
-    from("../${projectName}/${serviceName}/src/main/resources/configs/") {
+    from("../${projectName}/${serviceName}/src/${srcFolder}/resources/configs/") {
     }
   }
 }
+
+tasks.test {
+  useJUnitPlatform()
+  dependsOn("copyServiceConfigsTest");
+  dependsOn("createCopySpecForSubJobTest");
+}
+
+tasks.register<Copy>("copyServiceConfigsTest") {
```

Comment:
Could you please briefly describe what this task does and why we need it?

Reply:
So hypertrace-ingester topology is composed of 4 apps, span-normalizer, raw-spans-grouper, trace-enricher & view-generator. For running/testing this application we need all these config file under resource/config. 
This task basically copies these file under build...test/resource/config, I could instead manually added test config files, but that would always need changes in 2 places (service & ingester)

Reply:
Thanks for the explanation @rish691. Do you mind adding such description in the code?

Reply:
Done